### PR TITLE
Fix allowing uploading files bigger than limit

### LINKED ISSUE #81: Crash on video upload > 20MB
## What did you do?

I uploaded a video and app crashed. I see in logs, body size exceeded the limit (20M)
## What did you expect to happen?

It should not crash instead should throw error , if possible before even upload begins.

## What happened instead?
Crashed with

Thread 1: Fatal error: Binding error: responseError(StreamChatCore.ClientErrorResponse(code: 22, message: "body size exceeded the limit (20M)", statusCode: 400))
```

🐴 ⏫ [7] 32768/78055962, 0.0%
🐴 ⏫ [7] 65536/78055962, 0.0%
🐴 ⏫ [7] 98304/78055962, 0.0%
🐴 ⏫ [7] 131072/78055962, 0.0%
🐴 ⏫ [7] 163840/78055962, 0.0%
🐴 ⏫ [7] 229376/78055962, 0.0%
🐴 ⏱ Response received: 1.853 +1.853
🐴 ⬅️ Response 413 (95 bytes): https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPt/file?api_key=&user_id=&client_id=
🐴 📦 {
  "code" : 22,
  "message" : "body size exceeded the limit (20M)",
  "StatusCode" : 400,
  "duration" : "0.00ms"
}

```
## GetStream Environment
**GetStream Chat version:**  Stream Chat v.1.5.4
**iOS version:** 13
**Swift version:**  5
**Xcode version:**  11.3.1
**Device:** iPhone 7

### LINKED ISSUE #82: Crash on image upload
## What did you do?

I am using an example app , when I take photo from camera , it crashes because of network connectivity. 

## What did you expect to happen?
It should stop uploading image and should throw error instead of crashing. 
It would be great on such occasions if it would show restart button and resume upload. But nevertheless even if it doesn't crash that will be great. 

## What happened instead?

```Thread 1: Fatal error: Binding error: requestFailed(Optional(Error Domain=NSURLErrorDomain Code=-1005 "The network connection was lost." UserInfo={NSUnderlyingError=0x282a74ae0 {Error Domain=kCFErrorDomainCFNetwork Code=-1005 "(null)" UserInfo={NSErrorPeerAddressKey=<CFData 0x2806d4d20 [0x201a835e0]>{length = 16, capacity = 16, bytes = 0x100201bb3447a0070000000000000000}, _kCFStreamErrorCodeKey=-4, _kCFStreamErrorDomainKey=4}}, NSErrorFailingURLStringKey=https://chat-proxy-us-east.stream-io-api.com/channels/messagin```

## GetStream Environment
**GetStream Chat version:** Stream Chat v.1.5.4
**iOS version:** 13
**Swift version:** 5
**Xcode version:**  11.3
**Device:** iPhone7

## Additional context

Logs from Xcode Console 
```
🐴 Endpoint: sendEvent(StreamChatCore.EventType.typingStart, StreamChatCore.Channel)
🐴 ⏱ Sending request...
🐴 ➡️ POST https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/event?api_key=&user_id=&client_id=
🐴 📦 Request Body {
  "event" : {
    "type" : "typing.start"
  }
}
🐴 ⏫ [8] 262144/1100317, 24.0%
🐴 ⏱ Response received: 0.274 +0.274
🐴 🙅‍♂️ A request was cancelled. NSError -999
2020-01-30 12:07:39.964136+0530 Werq Chat[342:16137] Task <25D723B0-D5A0-4CF9-B173-20B599048477>.<9> finished with error [-999] Error Domain=NSURLErrorDomain Code=-999 "cancelled" UserInfo={NSErrorFailingURLStringKey=https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/event?api_key=&user_id=&client_id=, NSLocalizedDescription=cancelled, NSErrorFailingURLKey=https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/event?api_key=&user_id=&client_id=}
2020-01-30 12:07:39.969929+0530 Werq Chat[342:16478] Task <25D723B0-D5A0-4CF9-B173-20B599048477>.<9> HTTP load failed, 0/0 bytes (error code: -999 [1:89])
🐴 ⏫ [8] 294912/1100317, 27.0%
🐴 ⏫ [8] 327680/1100317, 30.0%
🐴 ⏫ [8] 360448/1100317, 33.0%
🐴 ⏫ [8] 393216/1100317, 36.0%
🐴 ⏫ [8] 425984/1100317, 39.0%
🐴 ⏫ [8] 458752/1100317, 42.0%
🐴 ⏫ [8] 491520/1100317, 45.0%
🐴 ⏫ [8] 524288/1100317, 48.0%
🐴 ⏫ [8] 557056/1100317, 51.0%
🐴 ⏫ [8] 589824/1100317, 54.0%
🐴 ⏫ [8] 622592/1100317, 57.0%
🐴 ⏫ [8] 655360/1100317, 60.0%
🐴 ⏫ [8] 688128/1100317, 63.0%
🐴 ⏫ [8] 720896/1100317, 66.0%
🐴 ⏫ [8] 753664/1100317, 68.0%
🐴 ⏫ [8] 786432/1100317, 71.0%
🐴 ⏫ [8] 819200/1100317, 74.0%
🐴 ⏫ [8] 851968/1100317, 77.0%
🐴 ⏱ Response received: 6.676 +6.402
🐴 🤷‍♂️ The network connection was lost. NSError -1005
2020-01-30 12:07:46.365473+0530 Werq Chat[342:16478] Task <EA09EE84-E4C8-48D3-B646-145436E86B33>.<8> HTTP load failed, 852714/0 bytes (error code: -1005 [4:-4])
2020-01-30 12:07:46.366247+0530 Werq Chat[342:16121] Task <EA09EE84-E4C8-48D3-B646-145436E86B33>.<8> finished with error [-1005] Error Domain=NSURLErrorDomain Code=-1005 "The network connection was lost." UserInfo={NSUnderlyingError=0x282a74ae0 {Error Domain=kCFErrorDomainCFNetwork Code=-1005 "(null)" UserInfo={NSErrorPeerAddressKey=<CFData 0x2806d4d20 [0x201a835e0]>{length = 16, capacity = 16, bytes = 0x100201bb3447a0070000000000000000}, _kCFStreamErrorCodeKey=-4, _kCFStreamErrorDomainKey=4}}, NSErrorFailingURLStringKey=https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/image?api_key=&user_id=&client_id=, NSErrorFailingURLKey=https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/image?api_key=&user_id=&client_id=, _kCFStreamErrorDomainKey=4, _kCFStreamErrorCodeKey=-4, NSLocalizedDescription=The network connection was lost.}
🐴 ❌  Error Domain=NSURLErrorDomain Code=-1005 "The network connection was lost." UserInfo={NSUnderlyingError=0x282a74ae0 {Error Domain=kCFErrorDomainCFNetwork Code=-1005 "(null)" UserInfo={NSErrorPeerAddressKey=<CFData 0x2806d4d20 [0x201a835e0]>{length = 16, capacity = 16, bytes = 0x100201bb3447a0070000000000000000}, _kCFStreamErrorCodeKey=-4, _kCFStreamErrorDomainKey=4}}, NSErrorFailingURLStringKey=https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/image?api_key=&user_id=&client_id=, NSErrorFailingURLKey=https://chat-proxy-us-east.stream-io-api.com/channels/messaging/!members-HSeqPtTUswzmZdAvRk9eJHwRPonhXN0l8n_Esj7kFBQ/image?api_key=&user_id=&client_id=, _kCFStreamErrorDomainKey=4, _kCFStreamErrorCodeKey=-4, NSLocalizedDescription=The network connection was lost.} in parse(data:response:error:completion:)[255]
(lldb) 
```

### PR DESCRIPTION
**Pull request by b-onc**
Fixes #81, fixes #82

### CODE REVIEW

File: Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift [hunk 1]
```diff
@@ -70,7 +70,17 @@ extension UIViewController {
         present(imagePickerViewController, animated: true)
     }
     
-    func showImpagePickerAuthorizationStatusAlert(_ status: PHAuthorizationStatus) {
+    private func showImagePickerAlert(title: String, message: String) {
```
Comment (buh) [inline Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift]: I'm not sure that we should show the alert in this case.

Comment (b-onc) [reply Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift]: You mean if the file size exceeds the limit?

Comment (buh) [reply Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift]: Yes. How a customer can handle that in own way?

Comment (b-onc) [reply Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift]: Well they don't have the ability to handle status alert in their own way too...

Comment (buh) [reply Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift]: I think will be better if you will change it to  use`ViewController:show(errorMessage:)` then customers could override it.

Comment (b-onc) [reply Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift]: That makes sense 👍 Should I do the same for status alerts? `show(errorMessage:)` does not accept a title/message though

**Pull request merged**